test/e2e/create_test.go

```diff
@@ -482,7 +482,7 @@ var _ = Describe("Podman create", func() {
 		inspect.WaitWithDefaultTimeout()
 		data := inspect.InspectContainerToJSON()
 		Expect(data).To(HaveLen(1))
-		Expect(data[0].Config).To(HaveField("StopSignal", uint(15)))
+		Expect(data[0].Config).To(HaveField("StopSignal", "SIGTTIN"))
```

Comment:
I'm not sure where this came from? `SIGTTIN` is 21, which is `0x15`, but I still don't understand this change?

Reply:
I fix that, it was SIGTERM not SIGTTIN. The new api return signal as string not int.